src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -112,7 +105,7 @@ FilterVariable generateFilter(SnapshotBuilderDomainCriteria domainCriteria) {
             ancestorPointer,
             "ancestor_concept_id",
             new FieldVariable(
-                new FieldPointer(occurrencePointer, occurrenceTable.idColumnName()),
+                new FieldPointer(occurrencePointer, domainOption.getColumnName()),
```

Comment:
Oh, fair. Hmm... I think I want to punt that to a future tech debt ticket, cause there's more implications than just reading it from the `RelationshipModel` I think? Because we still need to register the table, and there has to be some sort of logic that says where to compare IDs/get the list of IDs.